Yoast/wordpress-seo
# Improved suggested plugins UX

## Summary

This PR can be summarized in the following changelog entry:

* Improves the messages and adds installation / activation links when appropiate.

## Test instructions

This PR can be tested by following these steps:

* Install ACF and/or AMP.
* Go to SEO -> Dashboard and notice (depending on which plugin(s) you installed in the previous step) that one or more new notifications have been added with an install link.
* Install the suggested plugins, activate it and check that the notification disappears.
* Deactivate the installed plugin.
* Notice that there is now a notification with an activation link.
* Click the activation link and determine that the plugin is successfully reactivated.
* Remove one of the suggested plugins and then remove AMP or ACF.
* Notice that there is only one notification still present for the plugin that still remains.

Fixes https://github.com/Yoast/wordpress-seo/issues/8104
